www-apps/phpsysinfo/phpsysinfo-3.3.0.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit webapp
+
+DESCRIPTION="A customizable PHP script that displays information about your system nicely"
+HOMEPAGE="https://rk4an.github.com/phpsysinfo/"
+SRC_URI="https://github.com/rk4an/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86"
+IUSE="apcupsd dmraid freeipmi hddtemp ipmitool ipmiutil iptables lm_sensors mdadm megactl nut quota smart snmp uptime"
+RDEPEND="
+	dev-lang/php[simplexml,xml,xsl(+),xslt(+),unicode]
+	virtual/httpd-php
+	apcupsd? ( sys-power/apcupsd )
```

Comment:
Why are you adding  runtime dependencies flag only? This is our policy violation, if you wanna let user know about optional dependencies please use the `optfeature` function from the `eutils` eclass

Reply:
Pretty simple. No one told me, that this is supposed to be wrong, when I initially commited this and my PR got merged. As requested, I've moved all optional deps now into optfeature.